e2e/support/config.js

```diff
@@ -29,6 +37,8 @@ const isQaDatabase = process.env["QA_DB_ENABLED"] === "true";
 const sourceVersion = process.env["CROSS_VERSION_SOURCE"];
 const targetVersion = process.env["CROSS_VERSION_TARGET"];
 
+const doNotFail = process.env.CYPRESS_DO_NOT_FAIL === "true";
+
```

Comment:
😳  what's this for?

https://www.npmjs.com/package/volkswagen ?

Reply:
It is passed to 
```
 cy.compareSnapshot({
      name: buildSnapshotName(name),
      retryOptions: { doNotFail },
    })
```

It is set to `true` when we want to update snapshots